xml/System.Buffers/ArrayBufferWriter`1.xml

```diff
@@ -203,9 +228,17 @@
       </Parameters>
       <Docs>
         <param name="sizeHint">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns a <see cref="T:System.Span`1" /> to write to that is at least a specified length.
+            </summary>
+        <returns>A span of at least <paramref name="sizeHint" /> in length. If <paramref name="sizeHint" /> is not provided or is equal to 0, some non-empty buffer is returned.</returns>
+        <remarks>
```

Comment:
There is an issue with how remarks were written in the source which means only one of them got ported.

We should add the rest:
https://github.com/dotnet/corefx/blob/97d5957fc71853961082e22873d926269b291b92/src/Common/src/System/Buffers/ArrayBufferWriter.cs#L142-L150

Same problem with GetMemory.